core/src/test/scala/org/apache/spark/scheduler/cluster/mesos/MesosCoarseGrainedSchedulerBackendSuite.scala

```diff
@@ -257,10 +277,10 @@ class MesosCoarseGrainedSchedulerBackendSuite extends SparkFunSuite
 
   test("honors unset spark.mesos.containerizer") {
     setBackend(Map("spark.mesos.executor.docker.image" -> "test"))
-
+    val offeredPorts = (31100L, 31200L)
```

Comment:
Why add port offers to all the tests?  For simplicity, I'd prefer to keep the existing tests as they were.  Not a blocker, though.